api/src/http/heartbeat.rs

```diff
@@ -0,0 +1,15 @@
+use axum::{extract::State, http::StatusCode};
+use axum_extra::extract::cookie::CookieJar;
+
+use crate::http::AppState;
+
+pub async fn heartbeat(jar: CookieJar, State(_): State<AppState>) -> Result<(), StatusCode> {
```

Comment:
Consume token from `Authorization` header, because our client is a CLI.

Reply:
Prefer call this endpoint of `events`, because it will receive events, and not heartbeat only.